roles/tests/test_user_models.py

```diff
@@ -49,8 +49,8 @@ class FooObject(object):
             def __init__(self, foo_var):
                 self.foo_var = foo_var
 
-        user_group2 = FooObject('bar')
-        self.assertFalse(user_group1.__eq__(user_group2))
+        foo_object = FooObject('bar')
```

Comment:
pylint now errors when it sees a variable reassignment with a type different than the original type. Also, I couldn't name it `baz` because that's a "forbidden identifier". :)